DTS-STN/canadian-dental-care-plan
# Updated federal benefits page for child flow

### Description
Updated the federal provincial benefits page for the child flow from the figma design

Removed duplicate confirm translations in apply-adult-child.json

### Related Azure Boards Work Items
n/a

### Screenshots (if applicable)
![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/155585182/0e1ec026-591c-47e6-92a1-e8cddee3bb40)

### Checklist

### Test Instructions

comment out line 67 and 68 (invariant check)

add placeholder data, to view page

`const child = {
    id: 'id',
    dentalBenefits: {
      hasFederalBenefits: true,
      federalSocialProgram: '758bb862-26c5-ee11-9079-000d3a09d640',
      hasProvincialTerritorialBenefits: true,
      provincialTerritorialSocialProgram: 'b3f25fea-a7a9-ee11-a569-000d3af4f898',
      province: '9c440baa-35b3-eb11-8236-0022486d8d5f',
    },
    dentalInsurance: true,
  };`

### Additional Notes
n/a
